### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by Alexadlu** [votchallenge/toolkit-legacy#260]
Experiment baseline
  Tracker fast_manet
    Sequence afterrain
      Repetition 1
CLIENT: Socket opened successfully on port 9090.
CLIENT: Creating process /home/luandong/anaconda3/envs/python27/bin/python  -c "import sys; sys.path.append('/home/luandong/code/Transcation_RT-MANet-2019-11-18/TIP2020_rebutall/vot-toolkit-master/tracker/examples/fast_manet/');sys.path.append('/home/luandong/code/Transcation_RT-MANet-2019-11-18/TIP2020_rebutall/vot-toolkit-master/native/trax/support/python'); import run_fastmanet"
CLIENT: Working directory is /tmp/tp428fd989_3c0e_4e5c_b5be_e0ff0b3131ba
CLIENT: Starting process
CLIENT: Setting up TraX with TCP socket connection
@@TRAX:hello "trax.name=" "trax.family=" "trax.image=path;" "trax.region=rectangle;" "trax.description=" "trax.version=3" "vot=python" "trax.channels=color;ir;"
CLIENT: Tracker process ID: 18145
CLIENT: Connection with tracker established.
@@TRAX:initialize "file:///home/luandong/code/VOT2019/adlu_vot/vot-toolkit/sequences/afterrain/color/00001v.jpg" "file:///home/luandong/code/VOT2019/adlu_vot/vot-toolkit/sequences/afterrain/ir/00001i.jpg" "147.0000,53.0000,24.0000,45.0000"
@@TRAX:state "147.0000,53.0000,24.0000,45.0000"
@@TRAX:frame "file:///home/luandong/code/VOT2019/adlu_vot/vot-toolkit/sequences/afterrain/color/00002v.jpg" "file:///home/luandong/code/VOT2019/adlu_vot/vot-toolkit/sequences/afterrain/ir/00002i.jpg"
THCudaCheck FAIL file=/opt/conda/conda-bld/pytorch_1535488076166/work/aten/src/THC/THCGeneral.cpp line=663 error=11 : invalid argument
/home/luandong/code/Transcation_RT-MANet-2019-11-18/TIP2020_rebutall/vot-toolkit-master/tracker/examples/fast_manet/model.py:423: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  pos_loss = -F.log_softmax(pos_score)[:,1]
/home/luandong/code/Transcation_RT-MANet-2019-11-18/TIP2020_rebutall/vot-toolkit-master/tracker/examples/fast_manet/model.py:424: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  neg_loss = -F.log_softmax(neg_score)[:,0]
/home/luandong/code/Transcation_RT-MANet-2019-11-18/TIP2020_rebutall/vot-toolkit-master/tracker/examples/fast_manet/tracker_vot.py:183: UserWarning: torch.nn.utils.clip_grad_norm is now deprecated in favor of torch.nn.utils.clip_grad_norm_.
  torch.nn.utils.clip_grad_norm(model.parameters(), opts['grad_clip'])
@@TRAX:state "147.4216,53.7837,23.9038,44.7368"
@@TRAX:frame "file:///home/luandong/code/VOT2019/adlu_vot/vot-toolkit/sequences/afterrain/color/00003v.jpg" "file:///home/luandong/code/VOT2019/adlu_vot/vot-toolkit/sequences/afterrain/ir/00003i.jpg"
@@TRAX:quit
CLIENT: Cleaning up.
CLIENT: Stopping logger thread
CLIENT: Trying to stop process using protocol.
CLIENT: Trying to terminate process nicely.
CLIENT: Flushing streams
CLIENT: Process should be terminated.
CLIENT: Stopping logger.
CLIENT: Tracker exited (stopped by signal 15)
CLIENT: Closing server socket.
      Tracker execution interrupted: Did not receive response.
Error using traxclient
Did not receive response.

Error in tracker_run (line 77)
    data = traxclient(tracker.command, callback, ...

Error in experiment_supervised (line 72)
    data = tracker_run(tracker, @callback, data);

Error in tracker_evaluate (line 57)
        [files, metadata] = experiment_function(tracker, sequence, directory, parameters, scan);

Error in workspace_evaluate>execute_iterator (line 128)
            tracker_evaluate(event.tracker, event.sequence, event.experiment);

Error in iterate (line 65)
            context = iterator(event, context);

Error in workspace_evaluate (line 93)
context = iterate(experiments, trackers, sequences, 'iterator', iterator, 'context', context);

Error in run_experiments (line 10)
workspace_evaluate(tracker, sequences, experiments);

Error in run (line 96)
evalin('caller', [script ';']);

Why is there an error in the third frame？
the first frame and the second frame can work normally
Does anyone know what could be the problem？

**Comment by lukacu** [comment]
Usually, this is related to an error in the tracker. The problem is that capturing the output of a crashed process is a bit tricky so it may seem in a log that nothing has happened. You may want to put some try-catch statements in the code and see if you can determine the cause.
